src/overlays/actors/ovl_En_Ex_Ruppy/z_en_ex_ruppy.c

```diff
@@ -43,117 +43,123 @@ const ActorInit En_Ex_Ruppy_InitVars = {
 static Vec3f D_80A0B358[] = { { 0.0f, 0.1f, 0.0f }, { 0.0f, 0.0f, 0.0f } };
 static Vec3f D_80A0B370[] = { { 0.0f, 0.01f, 0.0f }, { 0.0f, 0.0f, 0.0f } };
 
-#ifdef NON_MATCHING
-// Regalloc
 void EnExRuppy_Init(Actor* thisx, GlobalContext* globalCtx) {
     EnExRuppy* this = THIS;
-    s16 temp_v0;
-    f32 phi_f12;
-    f32 temp;
+    EnDivingGame* divingGame;
+    f32 temp1;
+    f32 temp2;
+    s16 temp3;
 
     this->unk_152 = this->actor.params;
-    // Index
-    osSyncPrintf(VT_FGCOL(GREEN) "☆☆☆☆☆ インデックス ☆☆☆☆☆ %x" VT_RST, this->unk_152);
+
+    osSyncPrintf(VT_FGCOL(GREEN) "☆☆☆☆☆ インデックス ☆☆☆☆☆ %x\n" VT_RST, this->unk_152); // "Index"
```

Comment:
we've always put translation comments on top and have told people to move them from the side before